src/components/JobConfiguration.vue

```diff
@@ -181,10 +182,13 @@ export default defineComponent({
             role: 'cancel'
           }, {
             text: this.$t('Skip'),
-            handler: () => {
+            handler: async () => {
               if (job) {
-                this.store.dispatch('job/skipJob', job).then(() => {
-                  showToast(translate("This job has been skipped"))
+                await this.store.dispatch('job/skipJob', job).then((resp) => {
+                  if (resp) {
+                    emitter.emit('updatePendingJobs');
```

Comment:
```suggestion
                    emitter.emit('jobUpdated');
```
We need to update all component that a job is updated they could take action if required. This way we are not making it specific to pipeline page, but making it a generic solution

Reply:
Sir, Updated code.